Cargo.toml

```diff
@@ -88,7 +88,7 @@ ra-ap-rustc_lexer = { version = "0.53.0", default-features = false }
 ra-ap-rustc_parse_format = { version = "0.53.0", default-features = false }
 ra-ap-rustc_index = { version = "0.53.0", default-features = false }
 ra-ap-rustc_abi = { version = "0.53.0", default-features = false }
```

Comment:
Do bump the other three as well otherwise we get some duplicate transitive dependencies